### DISCUSSION TITLE
[Script request] PrivateBin

### DISCUSSION THREAD

**Discussion by jwgn** [community-scripts/ProxmoxVE#212] [Request script]
### Application Name

PrivateBin

### Website

https://privatebin.info/

### Description

A minimalist, open source online pastebin where the server has zero knowledge of pasted data. Data is encrypted/decrypted in the browser using 256 bits AES.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [x] This is not a game-related request.

**Comment by MickLesk** [comment]
Why not wastebin?

https://community-scripts.github.io/ProxmoxVE/scripts?id=wastebin

**Reply by PoempelPinguin** [reply]
Can't see any advantages to wastebin =/

**Comment by jwgn** [comment]
From what I see, that one does not support client-side encryption via anchor links, only optional password protection.
